src/components/Integrations/Shopify/Shopify.js

```diff
@@ -61,35 +62,66 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
       </tbody>
     </table>
   );
+
+  const shopifyHeader = (
+    <>
+      <div className="block">
+        <h2>{_('shopify.header_title')}</h2>
+        <p>
+          <FormattedHTMLMessage id="shopify.header_subtitle" />
+        </p>
+      </div>
+      <hr />
+    </>
+  );
+
+  function useInterval(callback, delay) {
```

Comment:
Could you move this to another place to allow sharing it?

Reply:
i told her in `src/utils` looks better